host/src/c_api/c_api.cc

```diff
@@ -1204,7 +1212,9 @@ bool attest_remote_report(
     size_t remote_report_size,
     const uint8_t* data,
     size_t data_size) {
-
+#ifdef __ENCLAVE_SIMULATION__
```

Comment:
This is kind of a nit, but do we need these three lines? `attest_remote_report` isn't exposed to the user, and is only called in `verify_remote_report_and_set_pubkey()` if we're not in simulation mode. So these three lines seem a bit redundant, unless `attest_remote_report()` is called somewhere other than `verify_remote_report_and_set_pubkey()`.

Reply:
Hmm, you're right. We can fix this in a separate PR along with any other refactoring we might want to do. E.g. I also want to change the name of `verify_remote_report_and_set_pubkey` to simply `verify_remote_report`.

Reply:
Ok!